### PR TITLE
fix side of street due to score not being distance

### PR DESCRIPTION
**Pull request by kevinkreiser**
also dont allow `valhalla_run_route` to use bidirectional for pedestrian (unless non-trivial) just like the service

### CODE REVIEW

File: src/loki/search.cc [hunk 1]
```diff
@@ -25,7 +25,7 @@ constexpr float NODE_SNAP = 5.f;
 //during side of street computations we figured you're on the street if you are less than
 //5 meters (16) feet from the centerline. this is actually pretty large (with accurate shape
 //data for the roads it might want half that) but its better to assume on street than not
-constexpr float SIDE_OF_STREET_SNAP = 5.f;
+constexpr float SIDE_OF_STREET_SNAP = 25.f;
```
Comment (kevinkreiser) [inline src/loki/search.cc]: since it uses the square distance now i changed this to the square, wasnt worth a `sqrt`

File: src/loki/search.cc [hunk 2]
```diff
@@ -88,7 +88,7 @@ float tangent_angle(size_t index, const PointLL& point, const std::vector<PointL
 }
 
 bool heading_filter(const DirectedEdge* edge, const EdgeInfo& info,
-  const Location& location, const PointLL& point, float distance, size_t index) {
```
Comment (kevinkreiser) [inline src/loki/search.cc]: this computation didnt even use distance

File: src/loki/search.cc (hunk 1 above)
Comment (dgearhart) [inline src/loki/search.cc]: do the comments need to be updated?

File: src/valhalla_run_route.cc [hunk 3]
```diff
@@ -676,8 +676,6 @@ int main(int argc, char *argv[]) {
     PathAlgorithm* pathalgorithm;
     if (routetype == "multimodal") {
       pathalgorithm = &mm;
-    } else if (routetype == "pedestrian") {
-      pathalgorithm = &bd;
```
Comment (kevinkreiser) [inline src/valhalla_run_route.cc]: this is inline with what the service does as well. once we refactor these test programs to use the `worker` methods we'll not need to worry about this logic.

**Comment by dgearhart** [comment]
:shipit:

File: src/loki/search.cc [hunk 4]
```diff
@@ -379,21 +379,23 @@ struct bin_handler_t {
       if(!candidate.edge->forward())
         length_ratio = 1.f - length_ratio;
       //side of street
-      auto side = candidate.get_side(location.latlng_, score);
+      auto side = candidate.get_side(location.latlng_, candidate.sq_distance);
+      PathLocation::PathEdge path_edge{candidate.edge_id, length_ratio, candidate.point, score, side, get_reach(candidate.edge)};
```
Comment (kevinkreiser) [inline src/loki/search.cc]: i liked the way the node correlation made the edges ahead of the `if`s and then `std::move`d them into the `vector`s so i followed suite here

**Pull request merged**